### DISCUSSION TITLE
Allow disabling OEM unlocking

### DISCUSSION THREAD

**Discussion by TommyTran732** [Divested-Mobile/DivestOS-Build#44] [General] [answered]
I have successfully installed DivestOS 18.1 onto my OnePlus 7T, enrolled the custom AVB Key and relocked the bootloader :)

So far, everything works as expected. However, I cannot seem to find an option to disable OEM unlocking in the developer options area. I am not sure if you disable this on purpose or if it is a bug somewhere. Since verified appears to be working properly, I think having an option to disable bootloader unlocking would be nice.

**Comment by SkewedZeppelin** [comment, marked as answer]
This was fixed recently:
https://github.com/Divested-Mobile/DivestOS-Build/commit/8004a11c52a8307e0d5b6a0793c4a0250285feca

Just haven't been any new builds with it yet.

**Reply by TommyTran732** [reply]
Ah okay thank you!

**Comment by SkewedZeppelin** [comment]
All builds should have the toggle available if appropriate now.
